src/routes/webhook/webhook-receiver-post.ts

```diff
@@ -0,0 +1,74 @@
+import { BinaryLike, createHmac } from "crypto";
+import { Request, Response } from "express";
+import { validationResult } from "express-validator";
+import { getLogger } from "~/src/config/logger";
+import { GitHubServerApp } from "~/src/models/git-hub-server-app";
+import { WebhookContext } from "./webhook-context";
+
+export const WebhookReceiverPost = async (request: Request, response: Response): Promise<void> => {
+	const errors = validationResult(request);
+	if (!errors.isEmpty()) {
+		response.status(400).json({ errors: errors.array() });
+		return;
+	}
+	const logger = getLogger("webhook.receiver");
+	const eventName = request.headers["x-github-event"] as string;
+	const signatureSHA256 = request.headers["x-hub-signature-256"] as string;
+	const id = request.headers["x-github-delivery"] as string;
+	const uuid = request.params.uuid;
+	let webhookSecret: string = process.env.WEBHOOK_SECRET!;
+	const payload = request.body;
+	try {
+		if (uuid != "cloud") {
```

Comment:
Doesn't need to be now, but in the long run, I would like to avoid if/else blocks like this one and instead handle all webhooks the same, be they from cloud or server. In this case, we could create a GitHubApp entry in the database for the cloud app.

Reply:
In one of the discussions, it is decided to keep Github cloud related configuration as it is i.e. they remain in environment variables. That is why database table named as `GitHubServerApps`

Reply:
the uuid would never be set at `cloud` though.  It's either undefined (coud) or defined (ghe)

Reply:
As discussed with @rachellerathbone , webhhook url for github cloud contains `cloud` as uuid.